nautobot/core/templates/components/panel/body_wrapper_key_value_table.html

```diff
@@ -1,3 +1,5 @@
-<table class="table table-hover panel-body attr-table"{% if body_id %} id="{{ body_id }}"{% endif %}>
-    {{ body_content }}
-</table>
+<div class="collapse show" {% if body_id %} id="{{ body_id }}"{% endif %}>
```

Comment:
Wrapping this `table` in a `div` element makes collapsible `panel` transition much smoother.